### DISCUSSION TITLE
Are there any plans to open source the figma design files for mantine?

### DISCUSSION THREAD

**Discussion by PeanutQAQ** [mantinedev/mantine#4039] [Q&A]
First of all, thank you very much for open sourcing this component library, it's arguably the most beautiful component library I've ever seen and mantine provides a lot of useful packages.

For myself, as a front-end engineer, I want to use mantine to copy and implement a toy component library of my own to learn about component library development, and I want to use mantine as a copy object, so I would like to ask if mantine can open source the figma design file of mantine.

Translated with www.DeepL.com/Translator (free version)

**Comment by rtivital** [comment]
Mantine does not have any design files to open source. All design work is happening in browser.
